pkg/grpc/config/config.proto

```diff
@@ -93,7 +71,7 @@ message Route {
 
   string kubernetes_service_account_token = 26;
 
-  OutlierDetection outlier_detection = 35;
+  envoy.config.cluster.v3.Cluster envoy_opts = 36;
```

Comment:
How do we want to handle deprecating pbs between releases? \cc @calebdoxsey @travisgroth

Reply:
I think `outlier_detection` didn't make it into any of the releases and console is not yet referring to it.